File: question-bank/questions/twosum/full/twosum_full.rs
##CODE_HERE##

pub fn main() {
    let mut input = String::new();
io::stdin().read_to_string(&mut input).unwrap();

let mut it = input.split_whitespace();
let size_arr0: usize = it.next().unwrap().parse().unwrap();

let arr:Vec<i32> = Vec::new();
for i in 0..size_arr0 {
    arr.push(it.next().unwrap().parse::<i32>().unwrap());
}
let target = it.next().unwrap().parse::<i32>().unwrap();
let result: i32 ;
result = twoSum(arr, target);
println!("{}", result);

}

File: question-bank/questions/twosum/full/twosum_full.cpp
##CODE_HERE##

int main() {
    int size_arr0;
cin>>size_arr0;
vector<int> arr(size_arr0);
for (int i=0; i< size_arr0; i++) {
    cin>>arr[i];
}
int target;
cin>>target;
vector<int> result;
result = twoSum(arr, target);
for (int i = 0; i < result.size(); i++) {
    if (i > 0) cout << " ";
    cout << result[i];
}


    return 0;
}
